cadquery/occ_impl/exporters/svg.py

```diff
@@ -250,8 +247,16 @@ def getSVG(shape, opts=None):
         {
             "unitScale": str(unitScale),
             "strokeWidth": str(strokeWidth),
-            "strokeColor": str(strokeColor[0]) + "," + str(strokeColor[1]) + "," + str(strokeColor[2]),
-            "hiddenColor": str(hiddenColor[0]) + "," + str(hiddenColor[1]) + "," +  str(hiddenColor[2]),
+            "strokeColor": str(strokeColor[0])
+            + ","
+            + str(strokeColor[1])
+            + ","
+            + str(strokeColor[2]),
+            "hiddenColor": str(hiddenColor[0])
+            + ","
+            + str(hiddenColor[1])
+            + ","
+            + str(hiddenColor[2]),
```

Comment:
Yikes, black has not helped here. Maybe:

```suggestion
            "strokeColor": ",".join([str(x) for x in strokeColor]),
            "hiddenColor": ",".join([str(x) for x in hiddenColor]),
```

Reply:
Thanks @marcus7070 I committed that change. Much, much better. Let me know if you see anything else.